llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1757,16 +1757,8 @@ struct LLVM_ABI_FOR_TEST VPWidenSelectRecipe : public VPRecipeWithIRFlags,
     return getOperand(0);
```

Comment:
Worth documenting somewhere that widen select recipe follows the select instruction in supporting both a widen and a uniform condition.

Reply:
Added, thanks